src/java/fr/paris/lutece/plugins/forms/business/Form.java

```diff
@@ -104,6 +104,10 @@
     private boolean _bOneResponseByUser;
 
     private int _nCurrentNumberResponse;
+    
+    private boolean _bCaptchaStepInitial;
+    private boolean _bCaptchaStepFinal;
+    private boolean _bCaptchaRecap;
```

Comment:
If I understand correctly, the user can configure the form so that 3 captchas are required (at the begining, at the end and at the recap). That seems excessive. Maybe it would be better to convert these 3 booleans to one enum, and to only allow one captcha ?

Reply:
The back office user can effectively configure up to three captchas with this development. Providing 3 captchas on a form is truly excessive but in some cases, administrators may want to configure two captchas. It is therefore better to stay with the booleans in order to take into account these situations.